styles/main.scss

```diff
@@ -118,6 +121,230 @@ nav a {
   }
 }
 
+.home-hero {
+  z-index: -1;
+  width: 100%;
+  height: 600px;
+  object-fit: cover;
+  object-position: 40% 10%;
+}
+
+.home-hero-text-group {
+  bottom: 104px;
+  max-width: 1128px;
+  @media screen and (min-width: 1600px) {
+    margin-left: 10%;
+  }
+  h1 {
+    font-family: $font-family-secondary;
+    font-size: 60px;
+    @include media-breakpoint-down(xxl) {
```

Comment:
nice! the sizes will be a little bit off for this PR, I will set the md/xl/xxl etc size variables in a later change